src/coreclr/inc/corinfo.h

```diff
@@ -2712,6 +2712,9 @@ class ICorStaticInfo
             CORINFO_SIG_INFO*           sig,            /* IN */
             CORINFO_ARG_LIST_HANDLE     args,           /* IN */
             CORINFO_CLASS_HANDLE       *vcTypeRet       /* OUT */
+#if defined(TARGET_LOONGARCH64)
+            ,int *flags = NULL
```

Comment:
This needs comment that explains what these flags are for, what the expected values are, etc.

Reply:
see https://github.com/dotnet/runtime/pull/62893